components/nimbus/src/tests/test_enrollment.rs

```diff
@@ -1205,9 +1205,6 @@ fn test_evolver_feature_can_have_only_one_experiment() -> Result<()> {
         json!({ "text": "OK then", "number": 42})
     );
 
-    let string = serde_json::to_string(&enrolled_feature.feature.value).unwrap();
```

Comment:
This test was just redundant, as we test the same thing a few lines above without converting to a string